sunpy/spectra/spectrogram.py

```diff
@@ -1260,4 +1260,8 @@ def in_interval(self, start=None, end=None):
                     *list(map(int, end.split(":")))
                 )
             end = self.time_to_x(end)
+        if start:
+            start = int(start)
+        if end:
+            end = int(end)
```

Comment:
Maybe I am confused, but instead of these if statements, would it not just be possible to int(start):int(end) on the return statement below?